src/styles/common.module.scss

```diff
@@ -0,0 +1,135 @@
+@import "variables";
+@import "colors";
+
+/* Font Sizes */
+$font-size-list: 10, 12, 14, 16, 18, 20, 24, 32, 40;
+@each $size in $font-size-list {
+  .font#{$size} {
+    font-size: #{$size}px;
```

Comment:
Think the font sizes should be handled in the typography component, depending on the project, always would prefer sizes in `em` or `rem` to scale better with the system

Reply:
@jacksonjp I updated this PR that using `rem` for font-size. Please check again. Thanks